tests/test_soda_imgserv.py

```diff
@@ -19,21 +19,14 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-import unittest
 
-class DALUnitTest(unittest.TestCase):
-    """Unit tests for DAL service components
+def test_soda_1():
+    ret = False
+    assert(ret is True)
 
-    TODO: implement these tests for generic SODA
-    """
 
-    def test_1(self):
-        return
+def test_soda_2():
+    ret = False
```

Comment:
So these tests still don't really exist, but now they fail instead of pass?  Does this get run as part of the CI?

Reply:
Good catch - the intention is to update these tests once SODA operations ready for unit testing, which among other things, require dummy (small) FITS files for testing purpose. I've changed the code to reflect this.